pkgs/top-level/all-packages.nix

```diff
@@ -597,8 +597,6 @@ with pkgs;
 
   dec-decode = callPackage ../development/tools/dec-decode { };
 
-  devour-flake = callPackage ../tools/nix/devour-flake { };
```

Comment:
Shouldn't this get an alias entry?

Reply:
@SuperSandro2000 Note that I _removed_ (not added) `devour-flake` entirely in this PR (it is no longer necessary). What change are you proposing here exactly?

Reply:
That you add a throw to aliases.nix so that evaluation is not broken if someone for whatever reason used this.

Reply:
Ah, understood. It is very likely that nobody has been using `pkgs.devour-flake` directly. I added it in #244487 for use in the old `nixci`.